manifest/spark-operator.yaml

```diff
@@ -43,7 +43,6 @@ spec:
       - name: sparkoperator
         image: gcr.io/spark-operator/spark-operator:v2.3.0-v1alpha1-latest
         imagePullPolicy: Always
-        command: ["/usr/bin/spark-operator"]
```

Comment:
fyi, going to tweak this and use /entrpoint here and move the command into the arg list